# request
int8 id
LightControlType command
---
# response
LightControlType state